addons/l10n_es_edi_facturae/models/account_move.py

```diff
@@ -147,8 +174,7 @@ def _l10n_es_edi_facturae_get_corrective_data(self):
             tax_period = refunded_invoice._l10n_es_edi_facturae_get_tax_period()
 
             reason_code = self.l10n_es_edi_facturae_reason_code or '10'
-            reason_description = [label for code, label in self._fields['l10n_es_edi_facturae_reason_code'].selection
-                                  if code == reason_code][0]
+            reason_description = SPANISH_CREDIT_REASON_TYPE.get(reason_code, '10')
```

Comment:
```suggestion
            reason_description = SPANISH_CREDIT_REASON_TYPE[reason_code]
```
We likely prefer to fail when the reason_code is not in the dict. I don't think it's even possible, actually.